src/main/java/core/mate/academy/model/Excavator.java

```diff
@@ -5,7 +5,10 @@
  * Do not remove no-args constructor
  */
 public class Excavator extends Machine {
-    public Excavator() {
+    private String model;
+    public Excavator(String color , String name , String model) {
```

Comment:
The class is missing a no-args constructor. According to the task description, you should not remove the no-args constructor. Please add a no-args constructor to this class.